src/megaapi_impl.cpp

```diff
@@ -23066,61 +23056,49 @@ int MegaSyncEventPrivate::getType() const
     return type;
 }
 
-const char *MegaSyncEventPrivate::getPath() const
+const char* MegaSyncEventPrivate::getPath() const
 {
-    return path;
+    return path.get();
 }
 
 MegaHandle MegaSyncEventPrivate::getNodeHandle() const
 {
     return nodeHandle;
 }
 
-const char *MegaSyncEventPrivate::getNewPath() const
+const char* MegaSyncEventPrivate::getNewPath() const
```

Comment:
no need to be switching space and * around

Reply:
`const char*` is in our coding guideline, `const char *` is not so we should change it if we're making changes anyway.